### PR TITLE
Group 7: Solution to Language Detector

### PR DESCRIPTION
**Pull request by ivanzugnoni**
Team Members
- Amrutha Vasil
- Ivan Zugnoni

### CODE REVIEW

**Comment by ivanzugnoni** [comment]
I removed many files which were included be mistake in the version control, for example: venv files, and some C9 temp files. I also fixed failing tests adding Python3 support.

File: language_detector/main.py [hunk 1]
```diff
@@ -1,9 +1,75 @@
...
+            d[word] = 1
+    max_value = 0
+    for key, value in d.items():
+        #print key, value
+        if value >= max_value:
+            max_value = value
+            word = key
+
+    return word  #returns the most common word found
+    """Returns the detected language of given text.
```
Comment (martinzugnoni) [inline language_detector/main.py:54]: Seems that the documentation strings are located in the incorrect place. Usually they must be at the very beginning inside of the function/class they are describing.

File: tests/test_main.py [hunk 2]
```diff
@@ -70,3 +75,14 @@ def test_detect_language_german_with_our_language_specification(self):
...
         result = detect_language(self.texts["german"], LANGUAGES)
         self.assertEqual(result.lower(), 'german')
+
+    def test_stats(self):
+        """Should return percentages of words occurrences in text for each language"""
+        expected = {
+            'German': '51.55%',
+            'Spanish': '10.1%'
+        }
+        self.assertEqual(calculate_percentages(self.text_occurrences), expected)
```
Comment (martinzugnoni) [inline tests/test_main.py:85]: Awesome! One extra point for this feature :)

Comment (nickczj) [inline tests/test_main.py:85]: First time peer reviewing! Have nothing to add. Really cool how you included stats calculation!

**Pull request closed without merging**